app/src/main/res/layout/activity_send_sms_invitation.xml

```diff
@@ -17,41 +16,64 @@
         android:id="@+id/note"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:padding="8dp"
         android:text="noteText"
         android:textSize="18sp"
-        android:padding="8dp"
         tools:ignore="HardcodedText" />
 
 
     <EditText
         android:id="@+id/number"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:hint="Phone Number"
-        android:inputType="phone"
         android:layout_margin="8dp"
         android:autofillHints=""
+        android:hint="Phone Number"
+        android:inputType="phone"
         tools:ignore="HardcodedText" />
 
     <EditText
         android:id="@+id/message"
         android:layout_width="wrap_content"
         android:layout_height="100dp"
-        android:hint="Type message"
-        android:text="Hi, I want you to register on BigOwl application. Please download it on Google Play store and sign up"
-        android:gravity="center_vertical"
         android:layout_margin="8dp"
         android:autofillHints=""
+        android:gravity="center_vertical"
+        android:hint="Type message"
+        android:text="Hi, I want you to register on BigOwl application. Please download it on Google Play store and sign up"
         tools:ignore="HardcodedText,TextFields" />
 
-    <Button
-        android:id="@+id/send"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="send sms to invite"
-        android:layout_gravity="center_horizontal"
-        tools:ignore="HardcodedText" />
+        android:layout_below="@+id/text_view_schedule_response_confirmation"
```

Comment:
This is a linearlayout, so this line shouldn't be here.

Reply:
yes, done